src/extension/wps/doc/GOGODUCK_README.md

```diff
@@ -14,9 +14,11 @@ To build:
 $ mvn clean install
 ```
 
-To run:
+To build:
 ```
-$ mvn exec:java -Dexec.mainClass=au.org.emii.gogoduck.worker.Main
+1. Navigate to http://localhost:9091/web/?wicket:bookmarkablePage=:org.geoserver.wps.web.WPSRequestBuilder
+2. Choose process: gs:GoGoDuck
+3. Choose format: application/x-netcdf
 ```
 
 ### Running as a WPS service. (simulating a portal)
```

Comment:
Perhaps "Submitting a request from the command line" would be a better heading - with "Submitting a request from Geoserver" being a better heading than "To build" above.